### PR TITLE
[NUI][TCSACR-298] Add feature to support MultiWindow

### PR DESCRIPTION
**Pull request by tscholb**
### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR:  TSACR-298

### CODE REVIEW

File: docs/application/dotnet/tutorials/app-filtering.md [hunk 1]
```diff
@@ -168,6 +168,7 @@ The following tables show the available requirements for an application package.
 | `http://tizen.org/feature/opengles.version.3_0` | Specify this key, if the application requires OpenGL&reg; ES version 3.0. You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered. | 2.4   |
 | `http://tizen.org/feature/opengles.version.3_1` | Specify this key, if the application requires OpenGL&reg; ES version 3.1. You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered. | 3.0   |
 | `http://tizen.org/feature/opengles.version.3_2` | Specify this key, if the application requires OpenGL&reg; ES version 3.2. You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered. | 4.0   |
+| `http://tizen.org/feature/opengles.surfaceless_context ` | Specify this key, if the application requires the EGL extension of Surfaceless Context for OpenGL® ES. | 6.0   |
```
Comment (mijin85cho) [inline docs/application/dotnet/tutorials/app-filtering.md:171]: In the table, please follow the alphabetical order. I suggest moving this feature to the front of  `http://tizen.org/feature/opengles.version.1_1`

```suggestion
| `http://tizen.org/feature/opengles.surfaceless_context` | Specify this key, if the application requires the EGL extension of Surfaceless Context for OpenGL&reg; ES. | 6.0   |
```

@annie-samsung Do you have any comment for this update?

Comment (annie-samsung) [reply docs/application/dotnet/tutorials/app-filtering.md:171]: @mijin85cho, no further comments from my side.

Comment (mijin85cho) [reply docs/application/dotnet/tutorials/app-filtering.md:171]: @annie-samsung Thanks for checking.

File: docs/application/dotnet/tutorials/app-filtering.md [hunk 2]
```diff
@@ -164,6 +164,7 @@ The following tables show the available requirements for an application package.
 | `http://tizen.org/feature/nlp` | Specify this key, if the application requires the Natural Language Processing (NLP) API. | 5.0 |
 | `http://tizen.org/feature/oauth2` | Specify this key, if the application requires the OAuth 2.0 protocol feature. | 4.0 |
 | `http://tizen.org/feature/opengles.version.1_1` | Specify this key, if the application requires OpenGL&reg; ES version 1.1 at minimum. You can specify at most 1 OpenGL&reg; ES version. If you specify multiple versions, only the highest one is considered. | 2.2.1 |
+| `http://tizen.org/feature/opengles.surfaceless_context` | Specify this key, if the application requires the EGL extension of Surfaceless Context for OpenGL&reg; ES. | 6.0   |
```
Comment (mijin85cho) [inline docs/application/dotnet/tutorials/app-filtering.md:167]: Thanks for the update, but I think there was a mistake at the location. Would you please follow the alphabetical order?

 - `http://tizen.org/feature/oauth2`
 - `http://tizen.org/feature/opengles.surfaceless_context`
 - `http://tizen.org/feature/opengles.version.1_1`

**Review by mijin85cho — APPROVED** [review]
Thanks for the update.

**Pull request merged**